multiraft/multiraft.go

```diff
@@ -644,10 +644,7 @@ func (s *state) start() {
 	})
 }
 
-func (s *state) removePending(g *group, prop *proposal, err error) {
-	if prop == nil {
-		return
-	}
+func (prop *proposal) removeFromPending(g *group, err error) {
```

Comment:
two reasons: `s` was never used in this method, and having a pointer receiver makes it clearly (to me?) that it's being mutated